algorithms/unit_tests/TestStdAlgorithmsMinMaxElementOps.cpp

```diff
@@ -329,10 +329,10 @@ void std_algorithms_min_max_element_test::test_minmax_element_non_trivial_data(
     }
     {
       const auto result =
-          KE::minmax_element(exespace(), KE::cbegin(view), KE::cend(view));
+          KE::minmax_element(exespace(), KE::begin(view), KE::end(view));
```

Comment:
Why this change?

Reply:
If we were to do:
```cpp
const auto result = KE::minmax_element(exespace(), KE::cbegin(view), KE::cend(view));
```
then result would be a pair of iterators computed as:
```cpp
result.first = cbegin() + something
result.second = cbegin() + something
```
So when we compare `result` against the gold solution , see line 206:
```cpp
EXPECT_EQ(result - KE::begin(testedView), goldPair.first);
```
since we use `begin` here, this raises an issue (at least on some compilers) because we are computing a distance from a const iterator and a non-const iterator. 

So by doing:
```cpp
const auto result = KE::minmax_element(exespace(), KE::begin(view), KE::end(view)); // notice we drop 'c'
```
then everything is consistent and works fine.

Reply:
i don't know why for this only some compilers complained, in fact in the CI this was not flagged. 
This only showed up in intel builds that Nathan pointed out I think,